server/src/main/java/de/uftos/services/TimetableService.java

```diff
@@ -60,9 +166,329 @@ public Timetable getById(String id) {
    * @throws ResponseStatusException is thrown if the ID defined in the timetable parameter is
    *                                 already present in the database.
    */
-  public Timetable create(TimetableRequestDto timetable) {
-    // solving magic here
-    return null;
+  public Timetable create(TimetableRequestDto timetable) throws BadRequestException {
```

Comment:
Again, services methods should only throw ResponseStatusExceptions, as they are being interpreted by spring with http Status codes